packages/api/src/sequelize/migrations/2024-11-19_00_update-patient-add-opting-out.ts

```diff
@@ -0,0 +1,24 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+
+const patientTableName = "patient";
+const optingOutColumn = "opting_out";
```

Comment:
I think we should use a column name that indicates the state of the patient, not an action. And we should be more clear as to what it means... maybe `hie_opt_out`?